Pod/Classes/UI/NumberInputTextField.swift

```diff
@@ -164,15 +164,15 @@ public class NumberInputTextField: StylizedTextField {
      
      - parameter cardNumber: The card number which should be displayed in `self`.
      */
-    public func prefillInformation(cardNumber: String) {
-        let unformattedCardNumber = String(cardNumber.characters.filter({$0.isNumeric()}))
+    public func prefill(text: String) {
```

Comment:
Should be `prefill(_ text: String)` like in `DetailInputTextField` (at least it should be kept consistent between those two files).